test/e2e/source_apiserver_test.go

```diff
@@ -55,6 +56,16 @@ func TestSourceApiServer(t *testing.T) {
 	test.apiServerSourceCreate(t, r, "testapisource0", "Event:v1:true", "testsa", "svc:testsvc0")
 	test.apiServerSourceCreate(t, r, "testapisource1", "Event:v1", "testsa", "svc:testsvc0")
 
+	t.Log("list sources")
+	output := test.sourceList(t, r)
+	assert.Check(t, util.ContainsAll(output, "NAME", "TYPE", "RESOURCE", "SINK", "READY"))
+	assert.Check(t, util.ContainsAll(output, "testapisource0", "ApiServerSource", "apiserversources.sources.knative.dev", "svc:testsvc0"))
+	assert.Check(t, util.ContainsAll(output, "testapisource1", "ApiServerSource", "apiserversources.sources.knative.dev", "svc:testsvc0"))
+
+	t.Log("list sources in YAML format")
+	output = test.sourceList(t, r, "-oyaml")
+	assert.Check(t, util.ContainsAll(output, "testapisource1", "ApiServerSource", "Service", "testsvc0"))
+
```

Comment:
thanks for already adding E2E tests, but maybe we could later enhance them with some more cases:

* Invalid filter option
* Heterogenous list of different types with `-o yaml` (would involve to combine a pingsource + apiserver source creation)